app/pages/index.js

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import { Footer, Navigation, Button } from '@button-inc/bcgov-theme';
+import Menu from '../components/Menu';
+import JsonSchemaForm from '@rjsf/semantic-ui';
+import schema from '../schemas/schema';
+import uiSchema from '../schemas/uiSchema';
+
+const Title = styled.h1`
+  font-size: 50px;
+  color: ${({ theme }) => theme.colors.primary};
+`;
+
+const SFooter = styled(Footer)`
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+`;
+
+const SJsonSchemaForm = styled(JsonSchemaForm)`
+  max-width: 90% !important;
+  margin: auto !important;
+`;
+
+const handleSubmit = ({ formData }, e) => {
+  console.log(formData);
+};
+
+export default function Home({ formData }) {
+  return (
+    <>
+      <Navigation
+        header="main"
+        title="Welcome to Connectivity Intake"
+      />
+      <Title>Connectivity Intake Form</Title>
+      <SJsonSchemaForm
```

Comment:
This styled component is inheriting the Semantic UI styles for `json-schema-form` right? There should be an example of grabbing the BC styles from [@button-inc/form-schema](https://github.com/button-inc/service-development-toolkit/tree/develop/packages/form-schema)

Reply:
Can you show me what you're referring to here? Am i to ditch using json schema form in favor of the toolkit one?

Reply:
Yes let's please try to use [the npm package we have](https://www.npmjs.com/package/@button-inc/form-schema) that wraps `json-schema-form`.